logging.go

```diff
@@ -98,13 +98,11 @@ func appScopeLogger(
 	logger *logrus.Logger,
 	appName string,
 	envName string,
-	buildVersion string,
 ) *Logger {
 	entry := logger.WithFields(logrus.Fields{
-		"app":           appName,
-		"env":           envName,
-		"build_version": buildVersion,
```

Comment:
The build version is in every log message because when we start doing canary deployments, it is useful to check logs for one variant vs another.